README.md

```diff
@@ -337,6 +338,54 @@ zone.createVM(name, { os: 'ubuntu' }, function(err, vm, operation) {
 ```
 
 
+## Google Translate API
+
+- [API Documentation][gcloud-translate-docs]
+- [Official Documentation][cloud-translate-docs]
+
+**An API key is required for Translate.** See [Identifying your application to Google][api-key-howto].
+
+#### Preview
+
+```js
+var gcloud = require('gcloud');
+
+var translate = gcloud.translate({
+  key: 'API Key'
+});
+
+// Translate a string of text.
+translate.translate('Hello', 'es', function(err, translation) {
```

Comment:
Is there a way to set the origin language as well as the target language? Looks like this auto-detects the origin (as 'en') and then sends it back in the 'es' as the target.